tool/matomo/classes/tool/tool.php

```diff
@@ -41,6 +41,8 @@ public function get_tracking_code() {
         $template = new \stdClass();
         $template->siteid = $settings['siteid'];
         $template->siteurl = $settings['siteurl'];
+        $usepiwik = (isset($settings['piwikurl']) && !empty($settings['piwikurl']));
```

Comment:
I found that 'piwikurl' is a bit confusing in this case. What do you think about renaming it to  'piwikjsurl' or ''custompiwikjsurl''? or something similar